portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includeLoginNav.vm

```diff
@@ -16,7 +16,7 @@
 
             #if (${useBullhornAlerts})
             <div class="Mrphs-sitesNav__menuitem portal-bullhorns-buttons">
-                <a id="Mrphs-bullhorn" href="javascript:void(0);" title="${rloader.sit_academic_bullhorn_title}" aria-haspopup="true">
+                <a id="Mrphs-bullhorn" href="javascript:void(0);" title="${rloader.sit_academic_bullhorn_title}" aria-haspopup="true" aria-expanded="false">
```

Comment:
Is a JS change needed to update the new `aria-expanded` value to true when the menu is opened?

Reply:
Mike I fixed the view all sites - but the bullhorn UI code is kind of a mess so I did not expand the scope to include  a fix for that.  I would suggest a new JIRA targeted to someone who knows more about Bullhorns than me.  It is an easy fix - but there needs to be a test plan etc since bullhorn's UI is pretty dynamic based on notifications etc.  So filing another JIRA is the right answer here.

Reply:
To answer your question - yes - the aria-expanded *does* need to toggle for it to be correct.   Each of these top level things has their own JS in library